skimage/util/apply_parallel.py

```diff
@@ -68,13 +68,18 @@ def apply_parallel(function, array, chunks=None, depth=0, mode=None,
     depth : int, optional
         Integer equal to the depth of the added boundary cells. Defaults to
         zero.
-    mode : 'reflect', 'periodic', 'wrap', 'nearest', optional
-        type of external boundary padding
+    mode : {'reflect', 'symmetric', 'periodic', 'wrap', 'nearest', 'edge'}, optional
+        type of external boundary padding.
     extra_arguments : tuple, optional
         Tuple of arguments to be passed to the function.
     extra_keywords : dictionary, optional
         Dictionary of keyword arguments to be passed to the function.
 
+    Notes
+    -----
+    Numpy edge modes `symmetric`, `wrap` and `edge` are converted to the
```

Comment:
This change is not directly related to the interpolation name changes in this PR, I just noticed it when checking all of these modes.  This function calls [dask](https://github.com/ContinuumIO/dask) which has yet another set of names than either `ndimage` or `numpy.pad`.  I just updated it so any `numpy.pad` names that have an equivalent in `dask` are converted properly.